projects/demo/src/modules/directives/dropdown/examples/1/index.html

```diff
@@ -2,15 +2,30 @@
     tuiButton
     tuiChevron
     type="button"
+    [attr.aria-expanded]="open()"
     [tuiDropdown]="dropdownContent"
-    [tuiDropdownManual]="open"
-    [tuiObscuredEnabled]="open"
+    [tuiDropdownManual]="open()"
+    [tuiObscuredEnabled]="open()"
     (click)="onClick()"
     (tuiActiveZoneChange)="onActiveZone($event)"
     (tuiObscured)="onObscured($event)"
 >
-    Choose
+    {{ buttonLabel() }}
     <ng-template #dropdownContent>
-        <div class="dropdown">But there is nothing to choose...</div>
+        <tui-data-list>
```

Comment:
I am not sure that we need to increase complexity of this example.
The main point of this first basic example (of `Dropdown` documentation page) to demonstrate how to use `[tuiDropdown]` directive. Not `Datalist`, not `Cell`.

I think, the such example should be added to `Datalist` documentation page

Reply:
just cherry pick

https://github.com/taiga-family/taiga-ui/pull/12412